server/bleep/src/semantic.rs

```diff
@@ -467,10 +468,28 @@ impl Semantic {
         branches: &[String],
         is_cold_run: bool,
     ) {
-        let chunk_cache =
-            crate::cache::ChunkCache::for_file(&self.qdrant, tantivy_cache_key, is_cold_run)
-                .await
-                .expect("qdrant error");
+        let chunk_cache = 'cache: {
+            let rand = rand::distributions::Uniform::new(500, 1500);
```

Comment:
Why is the backoff random? Can you add a comment explaining what's going on here and why